presto-docs/src/main/sphinx/release/release-331.rst

```diff
@@ -0,0 +1,83 @@
+* Enable dynamic filtering by default. (:issue:`2793`)
+* Show reorder join cost in ``EXPLAIN ANALYZE VERBOSE`` (:issue:`2725`)
+* Allow configuring resource groups selection based on user's groups. (:issue:`3023`)
+
+Security Changes
+----------------
+
+* Remove unused ``internal-communication.jwt.enabled`` configuration property. (:issue:`2709`)
+* Rename JWT configuration properties from ``http.authentication.jwt.*`` to ``http-server.authentication.jwt``. (:issue:`2712`)
+* Add access control checks for query execution, view query, and kill query. This can be configured using
+  :ref:`query_rules` in the file based system access control. (:issue:`2213`)
+* Prevent displaying columns of tables that user has no privilege granted in file based access control. (:issue:`2925`)
+* Add ``ALTER SCHEMA ... SET AUTHORIZATION`` syntax that allows to change the owner of the schema. (:issue:`2673`)
+
+Web UI Changes
+--------------
+
+* Fix copying worker address to clipboard. (:issue:`2865`)
+* Fix copying of query ID to clipboard. (:issue:`2872`)
+* Fix display of data size values. (:issue:`2810`)
+* Fix redirect from ``/`` to ``/ui/`` when Presto is run behind a proxy. (:issue:`2908`)
+* Fix ``Prepared Query`` section for :doc:`/sql/prepare` queries. (:issue:`2784`)
+* Display physical input read rate in UI. (:issue:`2873`)
+* Add simple form based authentication. Users can be authenticated with either the LDAP or the file based
+  authentication services. (:issue:`2755`)
+* Add support for disabling the web ui.  The UI can be disabled with the ``web-ui.enabled`` configuration
+  property. (:issue:`2755`)
+
+CLI Changes
+-----------
+
+* Fix formatting of ``varbinary`` in nested data types. (:issue:`2858`)
+
+Hive Connector Changes
+----------------------
+
+* Fix incorrect results of metadata queries and reads from ``information_schema`` tables
+  where connecting to Hive 3 metastore. (:issue:`3008`)
+* Fix query failure when using Glue metastore and table's storage descriptor has no properties. (:issue:`2905`)
+* Fix deadlock when Hive caching is enabled with refresh TTL. (:issue:`2984`)
+* Respect ``bucketing_version`` table property when using Glue metastore. (:issue:`2905`)
+* Add support for bucket sort order in Glue when creating or updating table or partition. (:issue:`1870`)
+* Add support for Hive full ACID tables. (:issue:`2068`, :issue:`1591`)
+* Allow data conversion when reading decimal data from Parquet files and precision or scale in the file schema
+  is different from the precision or scale in partition (table) schema. (:issue:`2823`)
+* Add option to enforce that a filter on a partition key be present in the query. This can be enabled by setting the
+  ``hive.query-partition-filter-required`` config property or the ``query_partition_filter_required`` session property
+  to ``true``. (:issue:`2334`)
+* Allow selecting ``Intelligent-Tiering`` S3 storage class when writing data to S3. This can be enabled using
```

Comment:
```
* Allow selecting the ``Intelligent-Tiering`` S3 storage class when writing data to S3. This can be enabled by
  setting the ``hive.s3.storage-class`` configuration property to ``INTELLIGENT_TIERING``. (:issue:`3032`)